lumu/info.json

```diff
@@ -0,0 +1,461 @@
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_incidents",
+      "title": "Get Incidents",
+      "description": "Retrieves a paginated list of incidents for the company. The items are listed by the most recent.",
+      "category": "investigation",
+      "annotation": "get_incidents",
+      "enabled": true,
+      "output_schema": {
+        "items": [
+          {
+            "id": "",
+            "timestamp": "",
+            "statusTimestamp": "",
+            "status": "",
+            "contacts": "",
+            "adversaries": [],
+            "adversaryTypes": [],
+            "labelDistribution": {
+              "17": ""
+            },
+            "totalEndpoints": "",
+            "lastContact": "",
+            "unread": ""
+          }
+        ],
+        "paginationInfo": {
+          "page": "",
+          "items": ""
+        }
+      },
+      "parameters": [
+        {
+          "title": "From Date",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "datetime",
+          "name": "fromDate",
+          "description": "Specify the Search start date. The default value is 7 days before the current date. Example: 2024-04-01T14:40:14.939Z",
+          "tooltip": "Specify the Search start date. The default value is 7 days before the current date. Example: 2024-04-01T14:40:14.939Z"
+        },
+        {
+          "title": "To Date",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "datetime",
+          "name": "toDate",
+          "description": "Specify the Search end date. The default value is the current date. Example: 2021-04-07T14:40:14.939Z",
+          "tooltip": "Specify the Search end date. The default value is the current date. Example: 2021-04-07T14:40:14.939Z"
+        },
+        {
+          "title": "Incident Status",
+          "name": "status",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "multiselect",
+          "options": [
+            "open",
+            "muted",
+            "closed"
+          ],
+          "tooltip": "Specify the Incident status. If not specified, all objects are returned.",
+          "description": "Specify the Incident status. If not specified, all objects are returned."
+        },
+        {
+          "title": "Adversary Type",
+          "name": "adversary-types",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "multiselect",
+          "options": [
+            "C2C",
+            "Malware",
+            "DGA",
+            "Mining",
+            "Spam",
+            "Phishing"
+          ],
+          "tooltip": "Specify the Adversary types. If not specified, all objects are returned.",
```

Comment:
Pls update tooltip and description:

Select one or more adversary types from the following options to filter incidents: C2C, Malware, DGA, Mining, Spam, or Phishing. If not specified, all objects are returned.